lib/components/user/places/place-editor.js

```diff
@@ -113,20 +115,32 @@ class PlaceEditor extends Component {
           <FlexFormGroup validationState={errorStates.address}>
             <PlaceLocationField
               className='form-control'
-              inputPlaceholder={isFixed ? `Search for ${place.name} location` : 'Search for location'}
+              inputPlaceholder={intl.formatMessage(
+                {
+                  id: `components.PlaceEditor.${isFixed ? 'locationPlaceholder' : 'genericLocationPlaceholder'}`
```

Comment:
Please do not create dynamic message IDs as it is hard to automatically keep track of which IDs are in the code. Instead, please create a hardcoded message ID string for each possible message.